packages/api/test/utils/checkAgainstSpec.ts

```diff
@@ -199,12 +199,24 @@ function prettyAjvErrors(errors: ErrorObject[] | null | undefined): string {
   return errors.map((e) => `${e.instancePath ?? "."} - ${e.message}`).join("\n");
 }
 
+type StringifiedProperty = string | StringifiedProperty[] | null | undefined;
+
+function stringifyProperty(value: unknown): StringifiedProperty {
+  if (value === undefined || value === null || typeof value === "string") {
```

Comment:
why do we need to handle `undefined` and `null`? we should ideally pass actual values in our test data

Reply:
Was needed for a previous version of the code.